apps/web/src/transitions.css

```
@@ -0,0 +1,249 @@
+    opacity: 1;
+    filter: blur(0);
+  }
+  to {
+    transform: translate3d(-12%, 0, 0) scale(0.96);
+    opacity: 0;
+    filter: blur(4px);
+  }
+}
+
+@keyframes route-rise-from-right {
+  from {
```

Stylelint (17.9.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)